src/drawing/DrawingPath.tsx

```diff
@@ -0,0 +1,95 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import { v4 as uuidv4 } from 'uuid';
+import { white } from 'box-ui-elements/es/styles/variables';
+import { DrawingSVGRef } from './DrawingSVG';
+import { PathGroup } from '../@types';
+import { Point } from '../region/transformUtil';
```

Comment:
Should we consider moving this to somewhere common instead of referencing `region` code in `drawing`?